app/domain/util/log_message_with_code_class.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'util/log_message_class'
+
+# A factory for creating a LogMessage with a code prefix
+#
+module Util
+  class LogMessageWithCodeClass
+    def self.new(msg:, code:)
+      LogMessageClass.new("#{code} #{msg}")
+    end
+  end
+end
```

Comment:
Small naming point:

What do you think about the name `TrackableLogMessageClass`?

`WithCode` is alright, but it reads a little clunky to me.  And the final purpose of the code is to allow the messages to be tracked and easily found (I think).

Reply:
nice